analyzer/src/main/kotlin/managers/CocoaPods.kt

```diff
@@ -192,14 +192,14 @@ class CocoaPods(
         )
     }
 
-    private fun getPodspec(id: Identifier, workingDir: File): Podspec {
+    private fun getPodspec(id: Identifier, workingDir: File): Podspec? {
         podspecCache[id.name]?.let { return it }
 
         val podspecName = id.name.substringBefore("/")
 
         val podspecFile = run(
             "spec", "which", podspecName, "--version=${id.version}", "--allow-root", "--regex", workingDir = workingDir
-        ).requireSuccess().stdout.trim().let { File(it) }
+        ).takeIf { it.isSuccess }?.stdout?.trim()?.let { File(it) } ?: return null
```

Comment:
This is becoming a bit unreadable, IMO. How about splitting this like

        val podspecCommand = run(
            "spec", "which", podspecName, "--version=${id.version}", "--allow-root", "--regex", workingDir = workingDir
        ).takeIf { it.isSuccess } ?: return null

        val podspecFile = File(podspecCommand.stdout.trim())